api/prisma/schema.prisma

```diff
@@ -145,6 +145,145 @@ model user {
 
 // -----------------------------------
 
+/// An exam as designed by the examiners
+model NewExam {
+  /// Globally unique exam id
+  ///
+  /// NOTE: Field is not "default", because of when/where exams are created.
```

Comment:
Could you explain why not? It's not clear from context why this is special.

Reply:
I could, but, now, I cannot 😅 

I clearly had a good reason for this at some point, but it has escaped me.